config/src/main/java/org/springframework/security/config/annotation/web/configurers/SessionManagementConfigurer.java

```diff
@@ -630,12 +630,8 @@ private <T> T getBeanOrNull(Class<T> type) {
 		if (context == null) {
 			return null;
 		}
-		try {
-			return context.getBean(type);
-		}
-		catch (NoSuchBeanDefinitionException ex) {
-			return null;
-		}
+
+		return context.getBeanProvider(type).getIfUnique();
```

Comment:
Same comment about `getIfAvaialble` here

Reply:
It's also same `getIfUnique` comment